diff --git a/CMakeLists.txt b/CMakeLists.txt
index af00701..0d15d75 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,6 +47,7 @@ find_package( duneprototypes REQUIRED EXPORT )
 find_package( dunecalib REQUIRED EXPORT )
 find_package( nuevdb REQUIRED EXPORT )
 find_package( duneopdet REQUIRED EXPORT )
+find_package( dunereco REQIRED EXPORT )
 find_package( dunesim REQUIRED EXPORT )
 find_package( larcoreobj REQUIRED EXPORT )
 find_package( larcore  REQUIRED EXPORT )
